Update readme (#62)
quantumblacklabs/kedro-examples: README.md

# What is `kedro-examples`?

> This repo contains links to all official and community created example Kedro projects as well as the complete version of the [Space Flights tutorial](https://kedro.readthedocs.io/en/stable/03_tutorial/02_tutorial_template.html).

Reviewer: ```suggestion
This repo contains links to all official and community created example Kedro projects as well as the complete version of the [spaceflights tutorial](https://kedro.readthedocs.io/en/stable/03_tutorial/02_tutorial_template.html).
```

In the Kedro docs we refer to it as "spaceflights" one word, and typically don't capitalise it.

> This repo contains links to all official and community created example Kedro projects as well as the complete version of the [Space Flights tutorial]((https://kedro.readthedocs.io/en/stable/03_tutorial/02_tutorial_template.html)).

Reviewer 2: Or are these double parentheses there for a reason? 🙂 
```suggestion
This repo contains links to all official and community created example Kedro projects as well as the complete version of the [Space Flights tutorial](https://kedro.readthedocs.io/en/stable/03_tutorial/02_tutorial_template.html).
```

Author: ah no that's a mistake, good catch!

  - [Kedro 0.15.8](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.15.8)
  - [Kedro 0.16.1](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.16.1)
  - [Kedro 0.16.4](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.16.4)

## Where can I find more learning material and `kedro-examples`?

The [`Kedro-Training`](https://github.com/quantumblacklabs/kedro-training) repository contains documentation that you can use to deliver Kedro training.

These repositories contain the official Kedro tutorial projects, also known as starters:

> - [Space Flights tutorial](https://github.com/quantumblacklabs/kedro-starter-spaceflights)

Reviewer: ```suggestion
- [Spaceflights tutorial](https://github.com/quantumblacklabs/kedro-starter-spaceflights)
```

Capitalised here because it's at the start of a bullet :)
